blueocean-core-js/src/js/utils/UrlUtils.js

```diff
@@ -34,7 +34,10 @@ export const buildPipelineUrl = (organization, fullName, tabName) => {
 
     return tabName ? `${baseUrl}/${tabName}` : baseUrl;
 };
-
+export const rootPathCreate = () => {
+    const jenkinsUrl = AppConfig.getJenkinsRootURL();
+    return `${jenkinsUrl}${jobPrefixPath(AppConfig.getOrganizationGroup())}/newJob`;
+};
```

Comment:
Could we name this method differently? I find it really helpful to include "classic" in the method names that creates URL's for classic UI. Perhaps `buildClassicCreateJobUrl` ?